bash/clean_sort_compress_data.sh

```diff
@@ -44,10 +44,10 @@ for file in $(find ${src_dir} -maxdepth 1 -iname 'omop_*.csv' -not -iname '*no_q
     # Set which field is the study ID
     case ${basename} in
         (omop_person_full.csv|bupropion*.csv|duloxetine*.csv)
-            key=1,1n # First field is study ID
+            keys="-k1,1n" # First field is study ID
             ;;
         (*)
-            key=2,2n # Second field is study ID
+            keys="-k2,2n -k1,1n" # Second field is study ID, first field is *_occurence_id
```

Comment:
Let's actually sort by date here rather than occurrence ID. I would also like to keep the long form of the options (more self-documenting, and amount of finger typing doesn't matter in scripts). So something like `keys="--key=2,2n --key=3,3"`.

Reply:
Maybe rename to `sort_keys` as well?